src/SOS/Strike/strike.cpp

```diff
@@ -1322,9 +1338,22 @@ HRESULT PrintVC(TADDR taMT, TADDR taObject, BOOL bPrintFields = TRUE)
     if ((Status=g_sos->GetMethodTableName(TO_CDADDR(taMT), mdNameLen, g_mdName, NULL))!=S_OK)
         return Status;
 
+    BOOL net9PreferCanonMT = FALSE;
```

Comment:
nit: Why not just eliminate this variable and use PreferCanonMTOverEEClassForNET9 directly below?

Reply:
I think Lee was suggesting not to have the `net9PreferCanonMT` flag and just call the function in the one place the result is needed.

Reply:
Sorry. I understand now.  Thanks!